### PR TITLE
Include Windows Server 20H2 image

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: https://deploy-preview-38371--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere#creating-the-vsphere-windows-vm-golden-image_creating-windows-machineset-vsphere

### CODE REVIEW

**Comment by codyhoag** [comment]
@sgaoshang @rrasouli can you verify? Thanks!

**Review by aravindhp — COMMENTED** [review]
/lgtm

File: modules/creating-the-vsphere-windows-vm-golden-image.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ You must use link:https://docs.microsoft.com/en-us/powershell/scripting/install/
 
 .Procedure
 
-. Create a new VM in the vSphere client using the Windows Server Semi-Annual Channel (SAC): Windows Server 2004 ISO image that includes the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. See the link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.hostclient.doc/GUID-FBEED81C-F9D9-4193-BDCC-CC4A60C20A4E.html[VMware documentation] for more information.
+. Create a new VM in the vSphere client using the Windows Server Semi-Annual Channel (SAC): Windows Server 2004 or 20H2 ISO image that includes the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. See the link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.hostclient.doc/GUID-FBEED81C-F9D9-4193-BDCC-CC4A60C20A4E.html[VMware documentation] for more information.
```
Comment (EricPonvelle) [inline modules/creating-the-vsphere-windows-vm-golden-image.adoc:21]: The link to the vSphere documentation results in a 404 error.

**Review by EricPonvelle — APPROVED** [review]
/lgtm

This looks good, but the VMWare documentation link needs to be fixed.

File: modules/creating-the-vsphere-windows-vm-golden-image.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/creating-the-vsphere-windows-vm-golden-image.adoc]: Thanks! Just updated it. :+1

**Pull request merged**